.github/workflows/deploy.yml

```diff
@@ -8,37 +8,119 @@ on:
     branches: [ master ]
 
 jobs:
-  deploy:
-    name: Deploy Demo App
+  deploy_chime_alpha_demo:
+    name: Alpha - Deploy Meeting Demo And Meeting Readiness Checker
```

Comment:
Minor one, but checking this is not readable until you extend 😅 :
![image](https://user-images.githubusercontent.com/61809205/146835292-f9fb9c4a-e66a-47bb-adcd-34f8391bebe6.png)

Suggest to update the name with some differentiator:
Beta - Chime Client -Deploy Meeting and Meeting Readiness Checker Demos
Beta - ChimeSDKMeetings Client -Deploy Meeting and Meeting Readiness Checker Demos

Reply:
updated